static/js/components/dashboard/courses/StatusMessages.js

```diff
@@ -86,6 +87,28 @@ const messageForNotAttemptedExam = (course: Course) => {
   return message
 }
 
+const courseStartMessage = (run: CourseRun) => {
+  if (run.course_start_date) {
+    return `Next course starts ${formatDate(run.course_start_date)}.`
+  } else if (run.fuzzy_start_date) {
+    return `Next course starts ${run.fuzzy_start_date}.`
+  }
+  return ""
+}
+
+const enrollmentDateMessage = (run: CourseRun) => {
+  if (
+    !R.isNil(run.enrollment_start_date) &&
+    !R.isEmpty(run.enrollment_start_date)
```

Comment:
can we make a utility for not null and not empty?

Reply:
you can use `!_.isEmpty(run.enrollment_start_dat)`. this function gives 
- `_.isEmpty('');` => true
- `_.isEmpty(null);` => true

Reply:
you can refactor it like
```js
const enrollmentDateMessage = (run: CourseRun) => {
   const enrollment_date =  !_.isEmpty(run. enrollment_start_date) ? formatDate(run.enrollment_start_date) : run.fuzzy_enrollment_start_date
   if (enrollment_date) {
       const startText = isEnrollableRun(run) ? "started" : "starts"
       return `Enrollment ${startText} ${enrollment_date}.`
   }
   return ""
}
```